### DISCUSSION TITLE
Package delivering

### DISCUSSION THREAD

**Discussion by dkositsyn** [PhilippeMarcMeyer/vanillaSelectBox#66] [Q&A]
Hey @PhilippeMarcMeyer, I found your library very nice, however it's hard to setup auto updates and delivering. Are you planning to publish it on npmjs, jsdelivr or similar resources?

**Comment by PhilippeMarcMeyer** [comment]
I must confess I don't know how to publish and I had no time to study this
I will study that and return to you ?

**Reply by PhilippeMarcMeyer** [reply]
I could use some help in the process...
On the other hand I would have to set the version to 1.0.0 ! which would need some refactoring and discussions
If you can spare somme time for me that would be great :)
